.gitignore

```diff
@@ -34,3 +34,5 @@ local
 
 gitinfo
 .envrc
+dataland-frontend/src/App.vue
+dataland-frontend/vite.config.ts
```

Comment:
Why are these files being excluded from version control?

Reply:
I added them before when I had to change them to run the whole stack on my local. I remove them now.